database/factories/ConferenceFactory.php

```diff
@@ -24,30 +24,52 @@ public function definition()
 
     public function closedCFP()
     {
-        return $this->state(function () {
-            return [
-                'cfp_starts_at' => $this->faker->dateTimeBetween('-9 days', '-4 day'),
-                'cfp_ends_at' => $this->faker->dateTimeBetween('-3 days', '-1 days'),
-            ];
-        });
+        return $this->state([
+            'cfp_starts_at' => $this->faker->dateTimeBetween('-9 days', '-4 day'),
+            'cfp_ends_at' => $this->faker->dateTimeBetween('-3 days', '-1 days'),
+        ]);
     }
 
     public function noCFPDates()
     {
-        return $this->state(function () {
-            return [
-                'cfp_starts_at' => null,
-                'cfp_ends_at' => null,
-            ];
-        });
+        return $this->state([
+            'cfp_starts_at' => null,
+            'cfp_ends_at' => null,
+        ]);
     }
 
     public function approved()
     {
-        return $this->state(function () {
-            return [
-                'is_approved' => true,
-            ];
-        });
+        return $this->state([
+            'is_approved' => true,
+        ]);
+    }
+
+    public function notApproved()
+    {
+        return $this->state([
+            'is_approved' => false,
+        ]);
+    }
+
+    public function shared()
+    {
+        return $this->state([
+            'is_shared' => true,
+        ]);
+    }
+
+    public function notShared()
+    {
+        return $this->state([
+            'is_shared' => false,
+        ]);
+    }
+
+    public function author($author)
```

Comment:
Similar to the `user` method in the `BioFactory`, I'd like to use Laravel's `for` method but calling `Conference::factory()->for($user, 'author')` everywhere is kinda clunky so I think we should keep this `author` method and change the implementation to `return $this->for($user, 'author')`.  Then all that will need to change is passing in `$user` instead of `$user->id`.